templates/hexagonal-spring-boot/domain-api/src/main/java/packageName/domain/model/ExampleInfo.java

```diff
@@ -0,0 +1,16 @@
+package packageName.domain.model;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class ExampleInfo {
```

Comment:
Should this class be part of rest-adapter? Later on while adding swagger documentation you have to again move it to rest-adapter considering domain purity.

Reply:
That rule is not just limited to REST. It can be applied and hence I think we will make it this way so that it is easier to scale

Reply:
I did not understand. Domain should be pure and it should be pure java without any dependency. If I want to configure this pojo with swagger documentation our domain will ultimately have swagger dependency. Am I missing something?

Reply:
We haven't configured anything specific to swagger ??